File: src/Domain/Entities/ToDoAssistant/Notification.cs
﻿using PersonalAssistant.Domain.Entities.Common;

namespace PersonalAssistant.Domain.Entities.ToDoAssistant
{
    public class Notification : Entity
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int ActionUserId { get; set; }
        public int? ListId { get; set; }
        public int? TaskId { get; set; }
        public string Message { get; set; }
        public bool IsSeen { get; set; }

        public User User { get; set; }
    }
}

File: src/Clients/to-do-assistant2/src/lib/models/viewmodels/notification.ts
export class Notification {
	constructor(
		public id: number,
		public listId: number,
		public taskId: number,
		public userName: string,
		public userImageUri: string,
		public message: string,
		public isSeen: boolean,
		public createdDate: string,
		public formattedCreatedDate: string,
		public url: string
	) {}
}
